### DISCUSSION TITLE
DED and PBF AM simulation in MOOSE

### DISCUSSION THREAD

**Discussion by dnajera27** [idaholab/moose#23089] [Q&A General] [answered]
I recently saw the announcement of MALAMUTE, an application focused on modeling and simulation of additive manufacturing processes. I was wondering whether this application is available and if not, when will it be released? And I was also wondering whether it will be able to simulate DED and PBF processes. 

https://inl.gov/article/simulation-code-probes-microscale-mysteries-of-advanced-manufacturing/

Thank you!

**Comment by GiudGiud** [comment]
@cticenhour

**Comment by cticenhour** [comment]
Hello, and thank you for your interest! MALAMUTE is focused on *advanced* manufacturing processes, not just additive, and we have some additive manufacturing capability coming online soon. Currently we can simulate aspects of electrothermal processes for electric field assisted sintering alongside laser-induced surface deformation and pool dynamics. DED capabilities will be coming online from several members of the development team this year - the points of contact for those efforts would be @dewenyushu and @jiangwen84 if they want to chime in. 

MALAMUTE is currently a licensed INL code, which means you should first contact the INL Technology Deployment office (see https://inl.gov/inl-initiatives/technology-deployment/) in order to obtain a license.

**Comment by cticenhour** [comment, marked as answer]
I wanted to announce an important change to the status of MALAMUTE - it is now open source! It can now be found at https://github.com/idaholab/malamute.
